Kudu.Services.Web/Pages/Index.cshtml

```diff
@@ -100,24 +101,30 @@
         <li>
             <a href="api/deployments">Deployments</a>
         </li>
-        <li>
-            <a href="api/scm/info">Source control info</a>
-        </li>
-        <li>
-            <a href="api/vfs">Files</a>
-        </li>
-        <li>
-            <a href="api/logs/docker">Current Docker logs</a> (<a href="api/logs/docker/zip">Download as zip</a>)
-        </li>
-    </ul>
-    <h3>Browse Directory</h3>
-    <ul>
-        <li>
-            <a href="deploymentlogs">Deployment Logs</a>
-        </li>
-        <li>
-            <a href="wwwroot">Site wwwroot</a>
-        </li>
+        @if (!Kudu.Core.Environment.IsOnLinuxConsumption)
```

Comment:
does cshtml support creating partial views (one for consumption and another for dedicated) to keep the logic isolated and reduce the ifs in this file.